patch/0001-tty-8250-HSUART-DMA-be-deactivated-for-DNV-CPU.patch

```diff
@@ -0,0 +1,45 @@
+From b63977b0e1421c641495940fafab9502a2a6d659 Mon Sep 17 00:00:00 2001
+From: Philo Shao <philo@micasnetworks.com>
+Date: Thu, 17 Apr 2025 02:17:18 -0500
+Subject: [PATCH] tty/8250: The HSUART DMA be deactivated for the DNV CPU.
```

Comment:
Remove *[PATCH]*, and use a summary/title in imperative mood, and no dot/period at the end.

> tty/8250: Deactivate HSUART for Intel DNV CPU